packages/react-server-test-pages/pages/navigation/playground.js

```diff
@@ -102,7 +102,7 @@ class FBL extends React.Component {
 	render() {
 		return <Link path={LINK(this.props.row)} frameback={true} {...this.props.link}>
 			<span className={this.state.available?'available':'not-available'}>FB</span>{
-				this.props.children?['/',...this.props.children]:[]
+				this.props.children?['/',...React.Children.toArray(this.props.children)]:[]
```

Comment:
Turns out this was turning each _character_ of the child text into a child component.  In React 0.14.x, this resulted in each character getting wrapped in `<span>`s, which was easy to miss, visually.  In React 15 each character got wrapped by a pair of HTML comments, which jumped right out. 😆